src/core/utils.tsx

```diff
@@ -35,7 +35,7 @@ export const removeEmptyArrays = (obj: null | undefined | Record<string | number
 };
 
 export const removeValuesFromObject = (obj: any, valuesToRemove: Array<string | null | undefined | number>) => {
-  const output = {};
+  const output: any = {};
```

Comment:
any?

Reply:
because it accepts any JSON object better limit input and output type with `Record<string, any>` to know it's an object
or you can use `unknown`

Reply:
Then we have to convert output everywhere that we have called this function. For example I want to call an API with input parameter of type `UpdateProfileReq`, but before calling API, I call this method to remove empty variables, the output would be of type `Record<string, any>`, again I have to cast this type to `UpdateProfileReq`, in some cases casting causes some more `ts` errors and it happens a lot in our code